vm_operator/config.yaml

```diff
@@ -1,14 +1,11 @@
 options:
-  app-env:
+  env:
     type: string
     default: "dev"
     description: "Application environment."
-  app-log-level:
+  log-level:
     type: string
     default: "info"
-  app-repo:
-    type: string
-    default: "https://github.com/matthew-hagemann/app-center-ratings"
   squid-proxy-url:
     type: string
-    default: "http://proxy.example.com"
+    default: ""
```

Comment:
I thought we were going to drop this and learn it from the JUJU_HTTP_PROXY env var?